qiime/split_libraries_lea_seq.py

```diff
@@ -103,7 +107,8 @@ def get_LEA_seq_consensus_seqs(fwd_read_f, rev_read_f,
                                min_reads_per_random_bc,
                                min_difference_clusters,
                                barcode_column,
-                               reverse_primer_column):
+                               reverse_primer_column,
+                               phred_offset):
```

Comment:
`phred_offset` isn't used anywhere, can you please remove from the library code and the script?